source/base/models.py

```diff
@@ -131,9 +131,10 @@ class EnvVar(models.Model):
     name = models.TextField(max_length=64)
     value = models.TextField(max_length=256, default='')
 
+# We are no longer having memex-explorer spin up additional containers dynamically.
+#Much of this is now dead code that can be removed. -- @amfarrell
```

Comment:
Feel free to take a scalpel to it when you get some time and you don't have higher priority tasks in your queue.